### ISSUE TITLE
[Bug]: On every boot, " draw over app" pop up

### ISSUE THREAD

**Issue by samadhanraut** [siddharthsky/CustTermux-JioTVGo#18]
### Problem description

On every boot on Firestick 3rd edition 2020, draw over app pop up appears,
Even after selecting ok, on next Boot , in  CustTermux-JioTVGo pop up appears again.
In initial  CustTermux-JioTVGo version it was not happening 
![IMG_20240819_083429](https://github.com/user-attachments/assets/3a86d840-8d3a-410c-8c6b-6c22aeeffec0)

### What steps will reproduce the bug?

On every boot 

### What is the expected behavior?

No pop up.
Earlier  CustTermux-JioTVGo version, such error was not happening 

### System information

```shell
Playlist url: http://localhost:5001/playlist.m3u

openssl/stable 1:3.3.1 arm [upgradable from: 1:3.2.1-1]

patch/stable 2.7.6-4 arm [upgradable from: 2.7.6-3]

readline/stable 8.2.13 arm [upgradable from: 8.2.10] all [ upgradable from: 3.12]

termux-keyring/stable 3.12-1 termux-tools/stable 1.43.3 all [upgradable from: 1.42.4]

termux-tools version:

1.42.4

Android version:

9

Kernel build information:

Linux localhost 4.4.162+ #1 SMP PREEMPT Sat Jul 6 19:14:23 UTC 2024 armv71 Android

Device manufacturer:

Amazon

Device model:

AFTSSS

LD Variables:

LD_LIBRARY_PATH=

LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so

~$
```

**Comment by Nexus-Gits** [comment]
Yes i also have face this issue. Please let me know how to fix this issue.

**Comment by siddharthsky** [comment]
This issue is fixed in CustTermux 4.2. 
https://github.com/siddharthsky/CustTermux-JioTVGo/releases/tag/x0.123.6
[Dismiss popup two times to disable it.]
